src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -860,6 +942,16 @@ private protected override void OnLoaded()
 	{
 		base.OnLoaded();
 
+
+		// WinUI has some special handling for ContentPresenter and ContentControl where even though they aren't Controls,
+		// they use OnApplyTemplate. As a workaround for now, we just call OnApplyTemplate here.
```

Comment:
Potentially related to https://github.com/unoplatform/uno/blob/9be0f64372ebd97f7dda6d4144d265d170c8d608/src/Uno.UI/UI/Xaml/FrameworkElement.Layout.crossruntime.cs#L192-L193